plugins/inputs/win_wmi/README.md

```diff
@@ -0,0 +1,280 @@
+# Windows Management Instrumentation Input Plugin
+
+This document presents the input plugin to read WMI classes on Windows
+operating systems. With the win_wmi plugin, an administrator is enabled to
+capture and filter virtually any configuration or metric value exposed through
+the Windows Management Instrumentation
+([WMI](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page))
+service.
+
+If telegraf is configured with a logfile and the plugin's configuration
+contains an invalid namespace, class, or property, an error is logged.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md
+
+## Configuration
+
+```toml @sample.conf
+# Input plugin to query Windows Management Instrumentation
+[[inputs.win_wmi]]
+  # specifies a prefix to attach to the measurement name
+  name_prefix = "win_wmi_"
+  [[inputs.win_wmi.query]]
+    # a string representing the WMI namespace to be queried
+    namespace = "root\\cimv2"
+    # a string representing the WMI class to be queried
+    class_name = "Win32_Volume"
+    # an array of strings representing the properties of the WMI class to be queried
+    properties = ["Name", "Capacity", "FreeSpace"]
+    # a string specifying a WHERE clause to use as a filter for the WQL
+    filter = 'NOT Name LIKE "\\\\?\\%"'

```

Comment:
Is there a doc explaining these types of filters and escaping values?

Reply:
I added a link to documentation.